src/components/gridList/types.ts

```diff
@@ -1,32 +1,35 @@
-import {FlatListProps} from 'react-native';
+import {FlatListProps, ScrollViewProps} from 'react-native';
 
 export interface GridListBaseProps {
   /**
    * Allow a responsive item width to the maximum item width
    */
-   maxItemWidth?: number;
-   /**
-    * Number of items to show in a row (ignored when passing maxItemWidth)
-    */
-   numColumns?: number;
-   /**
-    * Spacing between each item
-    */
-   itemSpacing?: number;
-   /**
-    * List padding (used for item size calculation)
-    */
-   listPadding?: number;
-   /**
-    * whether to keep the items initial size when orientation changes,
-    * in which case the apt number of columns will be calculated automatically.
-    */
-   keepItemSize?: boolean;
-   /**
-    * Pass when you want to use a custom container width for calculation
-    */
-   containerWidth?: number;
-
+  maxItemWidth?: number;
+  /**
+   * Number of items to show in a row (ignored when passing maxItemWidth)
+   */
+  numColumns?: number;
+  /**
+   * Spacing between each item
+   */
+  itemSpacing?: number;
+  /**
+   * List padding (used for item size calculation)
+   */
+  listPadding?: number;
+  /**
+   * whether to keep the items initial size when orientation changes,
+   * in which case the apt number of columns will be calculated automatically.
+   */
+  keepItemSize?: boolean;
+  /**
+   * Pass when you want to use a custom container width for calculation
+   */
+  containerWidth?: number;
+  /**
+   * Custom content container style
+   */
+  contentContainerStyle?: ScrollViewProps['contentContainerStyle'];
```

Comment:
Consider adding a `Pick<ScrollViewProps, 'contentContainerStyle'>` at the top instead, in any case this should probably be in the API.

Reply:
Fixed